schema/values/data-url.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "string",
+    "title": "Data URL",
+    "description": "An embedded data object",
+    "pattern": "^data:([\\w/]+)(;base64)?,(.+)$"
```

Comment:
I wonder if we should specify formats that need to be supported for these

Reply:
Yeah that would make sense. I'm also wondering if that is something we could/should have in the profile definition if there are differences in the formats players support.

Reply:
Might be worth documenting formats that we expect most players to support and then have profiles expand or reduce the set of formats supported.

Reply:
Since Skottie could be built without configurable image format support, I think it makes sense to have that specified in the profile and we can have some sample profiles about what we think a player should support. I did add a note about common formats to the doc page and that support may vary, but not sure if we want to go farther than that.